core/git_mixins/active_branch.py

```diff
@@ -95,22 +95,26 @@ def get_branch_status(self, delim=None):
         return status, secondary
 
     def get_branch_status_short(self):
-        detached, initial, branch, remote, clean, ahead, behind, gone = \
-            self._get_branch_status_components()
 
-        dirty = "" if clean else "*"
+        if self.in_rebase():
+            return "(no branch, rebasing {})".format(self.rebase_branch_name())
+        else:
```

Comment:
Since the `if` block above provides an early return, no need for all the rest of this to reside within an `else`.